style_index.css

```diff
@@ -36,26 +36,70 @@ body {
 	z-index: 1;
 }
 
+/* Style for search content - search system*/
+.search_content {
+	background: #ffb282;
+	opacity: ;
+	margin: 0vw 0vw 0vw 0vw;
+	padding: 3.7vw 0vw 0vw 0vw;
+	grid-column: 1 / span 2;
+	grid-row: 2;
+	z-index: 2;
+}
+
+/* Style for text input */
+input[type=text] {
+	width: 49vw;
+	height: 2.9vw;
+	margin: 3.4vw 0vw 0vw 0vw;
+  	border: none;
+	outline: none;
+	font-size: 1.1vw;
+	padding-left: 0.8vw;
+	padding-right: 0.8vw;
+	padding-top: 0.17vw;
+}
+
+
+/* Style for submit button */
+.button {
+	height: 3.21vw;
+	width: 6.5vw;
+  	border: none;
+	outline: none;
+	font-size: 1.1vw;
+	padding-left: 0.8vw;
+	padding-right: 0.8vw;
+	padding-top: 0.17vw;
+}
+
+.button:hover {
+	background-color: #424242;
+	transition: 0.23s ease-in-out;
+	cursor: pointer;
+}
+
 /* Style for space_1, which creates space between contents */
 .space_1 {
 	background: #fff;
 	opacity: ;
 	grid-column: 1 / span 2;
-	grid-row: 2;
+	grid-row: 3;
 	z-index: 2;
 }
 
+

```

Comment:
Below includes adjustments of margins, grid-template-columns and grid-template-rows.